### ISSUE TITLE
training only for keypoint detection without masks

### ISSUE THREAD

**Issue by bauerdavid** [Superlee506/Mask_RCNN#32]
Hello everyone,
I would like to train on my own dataset, but unfortunately it doesn't have mask annotations. Is it possible to train only for the keypoints? I had the idea of predicting the masks for my own images using the pretrained model so I could use it as input, but as I saw it for some random images the model didn't produce good results. This is because in my image set there are people who are making some acrobatic moves, and I guess COCO rather includes more usual postures (standing, sitting etc.).
Any help would be highly appreciated :)

**Comment by Superlee506** [comment]
@bauerdavid Sure you can train on your dataset withour mask. You just need to remove the mask loss and modify the dataloader.

**Comment by bauerdavid** [comment]
@Superlee506 Thank you for your answer! I will try it as soon as I get there.

**Comment by verewhittome** [comment]
@bauerdavid Also working on this - let me know how you get on!
